### DISCUSSION TITLE
GitHub notifications widget on iOS

### DISCUSSION THREAD

**Discussion by prometheas** [community/community#20227] [Mobile] [answered]
Hey there! Would really love to see the iOS/iPad Os apps add support for a notifications widget. Ideally with the ability to filter on specific organizations and/or repositories.

**Comment by elisealix22** [comment, marked as answer]
Hi @prometheas , thank you so much for the request! We have an open tracking issue on the iOS side with future ideas around widgets. I just added a note with your request! If you have any other additional feedback, please let us know.

**Reply by alialhammad1992** [reply]
in Saudi Arabia and add the Arabic GitHub language. Your audience is large there and we want that. We want to GitHub the and make it GitHub the Arabic language

**Reply by 0607912574** [reply]
Good morning people you are speaking with the king of the ages I need to name after me!!

**Reply by BlueFalconHD** [reply]
?
